src/css/main.css

```diff
@@ -1,7 +0,0 @@
-html, body, #cesiumContainer {
```

Comment:
Removing this css leaves a white border around the cesium viewer and causes the page to have a scrollbar, which looks pretty unpolished IMO. However, I see that this is also omitted from the [quickstart guide on the website](https://cesium.com/learn/cesiumjs-learn/cesiumjs-quickstart/). If the goal of this is to conform as closely to that tutorial as possible while using webpack, that's OK.